core/src/main/java/de/danoeh/antennapod/core/service/download/DownloadRequestCreator.java

```diff
@@ -31,6 +32,34 @@ public static DownloadRequest.Builder create(Feed feed) {
                 .lastModified(feed.getLastUpdate());
     }
 
+    public static DownloadRequest.Builder createTranscript(FeedMedia media) {
```

Comment:
Hmm, we had generic downloads like this some years ago (for images). I think this complicates quite a few things (especially if we want to gradually simplify the download logic, getting rid of `DownloadRequest`). Given that the transcripts are usually quite small files, how about just downloading it in `MediaDownloadedHandler`, where we also download the chapters?

Reply:
> how about just downloading it in `MediaDownloadedHandler`, where we also download the chapters?

Would that affect caching, or is that a separate thing?

Reply:
> Would that affect caching, or is that a separate thing?

@keunes It won't affect caching.

@ByteHamster I'm ready for another review.  I'll leave out any code for authentication until we see a need.